extensions/apple-reminders/src/menu-bar.tsx

```diff
@@ -163,7 +163,7 @@ export default function Command() {
   return (
     <MenuBarExtra
       isLoading={isLoading}
-      icon={{ source: { light: "icon.png", dark: "icon@dark.png" } }}
+      icon={{ source: { light: "menu-icon@light.png", dark: "menu-icon@dark.png" } }}
```

Comment:
```suggestion
      icon={{ source: { light: "menu-icon.png", dark: "menu-icon@dark.png" } }}
```

Reply:
You might need to change this 🙂

Reply:
Oh, somehow I missed this. Thanks, it's been fixed now. 😄